README.md

```diff
@@ -27,3 +27,9 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+The base case is n = 0 returns [0] when n is 1 it returns [0, 1]. So the array will have the correct number up to fib(n-1). So the invariant would be fib(n-1).
```

Comment:
This is not an invariant -- you can prove an invariant true. fib(n-1) is not a statement that you can prove true.